liblangutil/Token.h

```diff
@@ -44,6 +44,7 @@
 
 #include <iosfwd>
 #include <string>
+#include <cstdint>
```

Comment:
`uint8_t`, etc. types are not available without this header on `mingw` platform. It is required for Windows compilation.